DVOP-2123 v14 centos:8 to amazonlinux:2

autmwatl opened the pull request:
centos8 has been deprecated. migrate v14 to amazonlinux2

Commit:
DVOP-2123 v14 centos:8 to amazonlinux:2
centos8 has been deprecated. migrate v14 to amazonlinux2

Commit:
DVOP-2123 amend yum command
yum module not compatible with amazon linux. amend to install node directly.

sienkin commented on Dockerfile:
Line 5 is superfluous as `nodejs` is already included in the list of packages to be installed.

Commit:
DVOP-2123 remove duplicated install
remove installation of node as it is being installed in the subsequent yum install command

sienkin commented on Dockerfile:
The setup script installs a local repository configuration. As a result, it would be beneficial to have this executed before the `yum update -y && \` line. Best to have it on its own line as it's not directly related to the `yum` command. Similar to the way the v10 dockerfile is configured.

E.g.:
```
RUN curl --silent --location https://rpm.nodesource.com/setup_14.x | bash -

RUN yum update -y && \
```

Also, the `nss` package is installed by default in Amazon Linux 2 so can be omitted from the lsit of packages.

Commit:
DVOP-2123 Remove nss, separate commands
The nss package is installed by default in Amazon Linux 2. Omit from the list of packages to be installed.
Separate curl and update into separate execution lines for optimisation.

sienkin commented on Dockerfile:
Since this is a command on its own, you don't need the trailing `&& \`

Commit:
DVOP-2123 Remove unnecessary trailing &&
This is no additional command. Remove trailing &&
